docs: russian translation (#774)
t3-oss/create-t3-app: www/src/pages/ru/usage/prisma.md

```jsonc:package.json
{
  "scripts": {
    "db-seed": "NODE_ENV=development prisma db seed"
  },
  "prisma": {
    "seed": "tsx prisma/seed.ts"
  }
}
```

```ts:prisma/seed.ts
import { prisma } from "../src/server/db/client";

async function main() {
  const id = "cl9ebqhxk00003b600tymydho";
  await prisma.example.upsert({
    where: {
      id,
    },
    create: {
      id,
    },
    update: {},
  });
}

main()
  .then(async () => {
    await prisma.$disconnect();
  })
  .catch(async (e) => {
    console.error(e);
    await prisma.$disconnect();
    process.exit(1);
  });
```

Затем, просто запустите `pnpm db-seed` (или `npm`/`yarn`), чтобы заполнить вашу базу данных.

## Полезные ресурсы

| Ресурс                       | Ссылка                                                                                                                                            |
| ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |

> | Prisma Docs                  | https://www.prisma.io/docs/                                                                                                                       |

Reviewer: ```suggestion
| Документация Prisma                   | https://www.prisma.io/docs/                                                                                                                       |
```

| Prisma GitHub                | https://github.com/prisma/prisma                                                                                                                  |

> | NextAuth.JS Prisma Adapter   | https://next-auth.js.org/adapters/prisma                                                                                                          |

Reviewer: ```suggestion
| NextAuth.JS Адаптер для Prisma    | https://next-auth.js.org/adapters/prisma                                                                                                          |
```

> | Planetscale Connection Guide | https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/connect-your-database-typescript-planetscale |

Reviewer: ```suggestion
| Гайд по подключению PlanetScale | https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/connect-your-database-typescript-planetscale |
```
